src/fsharp/fsi/fsi.fs

```diff
@@ -204,16 +204,16 @@ module internal Utilities =
 [<AutoSerializable(false)>]
 type internal FsiTimeReporter(outWriter: TextWriter) =
     let stopwatch = Stopwatch()
-    let ptime = Process.GetCurrentProcess()
+    let ptime = lazy Process.GetCurrentProcess()
```

Comment:
Why is this changed to Lazy?  I don't really understand why it would be necessary?

Reply:
If it isn't, then that call is evaluated eagerly when `FsiTimeReporter` is constructed, even if its never actually used. I admit, it was more of a quick way to make it behave where `Process` isn't available, and it would probably be better to lazily construct `FsiTimeReporter` instead, but this was easier to find and solved `Process` throwing.

Reply:
I think I get it.  Is Process.GetCurrentProcess() particularly expensive?

Reply:
No, the problem is that on WASM it throws unconditionally. By making it lazy, it's never evaluated until its needed for timing, which is off by default.